FakeItEasy.build

```diff
@@ -1,240 +0,0 @@
-			XmlFileName="$(FxCopOutput)"
-			XPath="string(count(//Issue[@Level='Error']))">
-			<Output TaskParameter="Value" PropertyName="FxCop_Errors" />
-		</XmlRead>
-		<XmlRead ContinueOnError="True"
-			XmlFileName="$(FxCopOutput)"
-			XPath="string(count(//Issue[@Level='CriticalWarning']))">
-			<Output TaskParameter="Value" PropertyName="FxCop_CriticalWarnings" />
-		</XmlRead>
-		<XmlRead ContinueOnError="True"
-			XmlFileName="$(FxCopOutput)"
-			XPath="string(count(//Issue[@Level='Warning']))">
-			<Output TaskParameter="Value" PropertyName="FxCop_Warnings" />
-		</XmlRead>
-		<XmlRead ContinueOnError="True"
-			XmlFileName="$(FxCopOutput)"
-			XPath="string(count(//Issue))">
-			<Output TaskParameter="Value" PropertyName="FxCop_TotalErrors" />
-		</XmlRead>
-
-		<Warning Text="FxCop encountered $(FxCop_TotalErrors) rule violations"
-				Condition="$(FxCop_TotalErrors) &gt; 0" />
-
-		<Error Text="FxCop encountered $(FxCop_TotalErrors) rule violations"
-				Condition="$(FxCop_TotalErrors) &gt; 0" />
-	</Target>
-	
-	<Target Name="Package" DependsOnTargets="Clean;Rebuild;Merge">
-		<MakeDir Directories="$(NuPackPackageDirectory)\$(VersionNumber)\lib" />
-		<MakeDir Directories="$(NuPackPackageDirectory)\$(VersionNumber)\lib\NET40" />
-		<MakeDir Directories="$(NuPackPackageDirectory)\$(VersionNumber)\lib\SL4" />
-		
-		<CreateItem Include="$(BinariesPath)\UnMerged\FakeItEasy.*;">
-			<Output ItemName="fakeItEasyLibFiles" TaskParameter="Include"/>
-		</CreateItem>
-		
-		<CreateItem Include="$(BinariesPath)\UnMerged-SL\FakeItEasy-SL.*;">
-			<Output ItemName="fakeItEasyLibFiles-SL" TaskParameter="Include"/>
-		</CreateItem>
-		
-		<Copy SourceFiles="@(fakeItEasyLibFiles)" DestinationFolder="$(NuPackPackageDirectory)\$(VersionNumber)\lib\NET40" />
-		<Copy SourceFiles="@(fakeItEasyLibFiles-SL)" DestinationFolder="$(NuPackPackageDirectory)\$(VersionNumber)\lib\SL4" />
-		<Copy SourceFiles=".\Resources\Build\FakeItEasy.nuspec" DestinationFolder="$(NuPackPackageDirectory)\$(VersionNumber)" />
-		
-		<!-- Update version number in NuPack file -->
-		<XmlUpdate XmlFileName="$(NuPackPackageDirectory)\$(VersionNumber)\FakeItEasy.nuspec" Xpath="/package/metadata/version"  Value="$(VersionNumber)" />
-		
-		<!-- Create package file -->
-		<Exec 
-			Command=".\Resources\NuGet.exe pack $(NuPackPackageDirectory)\$(VersionNumber)\FakeItEasy.nuspec /o .\Build\Package\" 
-			ContinueOnError="False" />
-	</Target>	
-	
-	<Target Name="Distrib" DependsOnTargets="Clean;CodeAnalysis;ReBuild;Merge;Merge-SL;Package">
-		<MakeDir Directories="$(ArchiveDirectory)" />
-
-		<!-- Archive-->
-		<CreateItem Include="$(BinariesPath)\Merged\*.*;$(BinariesPath)\Merged-SL\*.*;;$(LicensePath)\*.*">
-			<Output ItemName="archiveFiles" TaskParameter="Include" />
-		</CreateItem>
-
-		<Zip Files="@(archiveFiles)" ZipFileName="$(ArchiveDirectory)\$(ProjectName)-$(VersionNumber).zip" Flatten="true" WorkingDirectory="$(BuildPath)" />
-	</Target>
-</Project>

```

Comment:
Wow. That monstrosity must be pretty old, I've never seen it before.

Reply:
I'm not saying I've never seen it, but I don't remember it!